pkg/credentials/aws.go

```diff
@@ -0,0 +1,33 @@
+package credentials
```

Comment:
Question: do you think it made sense to add some tests to this again?

Reply:
For functions like this I could go either way - let's err on the side of caution and add some tests.

Reply:
For functions like this, it feels weird to test because my tests would essentially just do exactly what the function is already doing. For example...

```
	type fields struct {
		AccessKeyID     string
		SecretAccessKey string
		SessionToken    string
		Region          string
		Expiration      string
	}
	tests := []struct {
		name   string
		fields fields
		want   string
	}{
		{
			name:   "Contains no values",
			fields: fields{},
			want:   fmt.Sprintf(AwsExportFormat, "", "", "", ""), <-- this is just what the function itself is doing
		},
	}
```
That said - I could still go either way. Any thoughts?